dbms/src/Common/FieldVisitors.cpp

```diff
@@ -221,57 +221,121 @@ String FieldVisitorToString::operator()(const Tuple & x_def) const
 
 String FieldVisitorToDebugString::operator()(const Null &) const
```

Comment:
Can we use template to reduce similar code?

Reply:
The `UInt64/Int64/DecimalField<...>` share the similar code, but other types are not.
We can try to eliminate it using `std::enable_if`, but it is not related to this PR change, maybe we can do it in another PR.